README.md

```diff
@@ -4,15 +4,49 @@ A cli of r utilities aimed at package and .Rprofile management.
 
 ## Dependenciecs
 
-`rut` depends on R and python3 ([poetry](https://poetry.eustace.io/docs/#installation) and [click]().
+`rut` depends on R and python3 ([poetry](https://poetry.eustace.io/docs/#installation) and [click](https://palletsprojects.com/p/click/)).
 
 ## Installation
 
+Currently, this project is not on pypi, but you can install using the below
+instructions if you have poetry installed.
 
+```console
+$ git clone https://github.com/datasnakes/rut.git
+$ cd rut
+$ poetry build
+$ pip install dist/rut-0.1.0-py3-none-any.whl
+```
 
 ## Usage
```

Comment:
@santina Thanks for the feedback! That does exist at https://github.com/datasnakes/rut/blob/850ed2ef11b372296df1bbaa64464f246a5fb9ea/README.md#using-rs-remotes-package but I do need to add the help message for that. I also will remove the jetpack usage example since those updates haven't been pushed here yet.